helix-term/src/commands/lsp.rs

```diff
@@ -496,6 +496,42 @@ pub fn code_action(cx: &mut Context) {
         },
     )
 }
+
+impl ui::menu::Item for lsp::Command {
+    type Data = ();
+    fn label(&self, _data: &Self::Data) -> Spans {
+        self.title.as_str().into()
+    }
+}
+
+pub fn workspace_command_picker(cx: &mut Context) {
+    let (_, doc) = current!(cx.editor);
+
+    let language_server = language_server!(cx.editor, doc);
+
+    let execute_command_provider = match &language_server.capabilities().execute_command_provider {
+        Some(p) => p,
```

Comment:
Very minor nit but I think this would be clearer as `options` since the type is `ExecuteCommandOptions`